server-sq/routes/queue.js

```diff
@@ -1,24 +1,45 @@
 const express = require("express");
 const router = express.Router()
-const SpotifyWebApi = require("spotify-web-api-node");
 
-var queue = [];
 
-router.get('/', (req, res) => {
-    res.send('Queue routing check')
-})
+module.exports = function(spotifyApi) {
+    var queue = []; // Will eventually translate to playback queue.
+    
+    
+    //async function filter(trackUri) {
+    //    let res = await spotifyApi.getAudioFeaturesForTrack(trackUri).then()
+    //    let features = res.body;
+    //    //console.log(features);
+    //    //Parse Features - REQUIRES UPDATE
+    //    //updated based on piano, classical music, and other slow/sleepy song track features (update if needed)
+    //    if (features.energy <= 0.3 || 
+    //        features.loudness <= -17 ||
+    //        features.acousticness >= .8 ||
+    //        features.instrumentalness >= 0.60 ||
+    //        features.valence <= 0.15 ||
+    //        features.tempo <= 80 ) {
+    //        return false;
+    //    }
+    //    return true;
+    //}
```

Comment:
We can completely remove this. Let's avoid putting comments that are no longer needed in production code.